toolchain-fitnesse-plugin/src/test/java/nl/praegus/fitnesse/responders/testHistory/RecentTestHistoryResponderTest.java

```diff
@@ -62,6 +63,24 @@ public void Checks_number_of_times_passed(){
 
         assertThat(recentTestHistory.getHistoryLines().get(0).getNumberOfTimesPassed()).isEqualTo(0);
     }
+    @Test
+    public void Checks_if_filter_is_off(){
```

Comment:
Als het filter uitstaat, dan krijgen we alle resultaten terug. Zoiets moet het toch testen?